webapp/src/components/calculations/calculations.ts

```diff
@@ -0,0 +1,180 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import {Card} from '../../blocks/card'
+import {IPropertyTemplate} from '../../blocks/board'
+import {Utils} from '../../utils'
+import {Constants} from '../../constants'
+
+const roundedDecimalPlaces = 2
```

Comment:
Probably wanna make this all caps

![image](https://user-images.githubusercontent.com/17804942/128706491-18663989-cf76-4f4d-a499-8662fe523a3a.png)